### PR TITLE
handle nans

### PR DESCRIPTION
**Pull request by jarry-xiao**

### CODE REVIEW

**Review by eugene-chen — COMMENTED** [review]
Let's definitely log something here, and perhaps try reconnecting

File: rust/phoenix-sdk/src/price_listeners/coinbase.rs [hunk 1]
```diff
@@ -128,6 +174,13 @@ impl CoinbasePriceListener {
                         },
                         Message::Ticker(ticker) => {
                             let price = ticker.price();
+                            if price.is_nan() || price.is_infinite() || *price <= 0.0 {
+                                println!(
+                                    "Price is invalid: {}, reconnecting as after 10 seconds",
```
Comment (eugene-chen) [inline rust/phoenix-sdk/src/price_listeners/coinbase.rs:179]: Does this reconnect?

File: rust/phoenix-sdk/src/price_listeners/coinbase.rs [hunk 2]
```diff
@@ -86,19 +86,65 @@ impl CoinbasePriceListener {
...
                                     .collect::<Vec<_>>();
 
                                 modified_ladder.update_orders(Side::Ask, update_asks);
+                                if !response_ok {
+                                    println!(
+                                        "Response is invalid, bids: {:?}, asks {:?}",
+                                        bids, asks
+                                    );
+                                    continue;
+                                }
```
Comment (eugene-chen) [inline rust/phoenix-sdk/src/price_listeners/coinbase.rs:147]: do this for L2update as well

File: rust/phoenix-sdk/src/price_listeners/coinbase.rs (hunk 1 above)
Comment (jarry-xiao) [reply rust/phoenix-sdk/src/price_listeners/coinbase.rs:185]: yeah, the loop is inside another loop

**Review by eugene-chen — APPROVED** [review]

**Pull request merged**